apps/docs/content/guides/platform/orioledb.mdx

```diff
@@ -0,0 +1,118 @@
+---
+id: 'orioledb'
+title: 'OrioleDB: a cloud-native storage engine for PostgreSQL'
+description: "A storage extension for PostgreSQL which uses PostgreSQL's pluggable storage system"
+---
+
+The [OrioleDB](https://www.orioledb.com/) Postgres extension provides a drop-in replacement storage engine for the default heap storage method. It is designed to improve Postgres' scalability and performance.
+
+OrioleDB is in active development and currently has [certain limitations](https://www.orioledb.com/docs/usage/getting-started#current-limitations). Currently, only B-tree indexes are supported, so features like pg_vector's HNSW indexes are not yet available. We are actively developing an Index Access Method bridge to unlock support for all index types used with heap storage. In Supabase OrioleDB image the default storage method has been updated to use OrioleDB by default, granting better performance out of the box.
+
+OrioleDB addresses PostgreSQL's scalability limitations by removing bottlenecks in the shared memory cache under high concurrency and optimizing WAL insertion through a row-level WAL loggin. These improvements are demonstrated by the TPC-C benchmark, which assess real-world database performance. The benchmark was performed on a c7g.metal instance and a 500-warehouse dataset and shows OrioleDB's performance compared to default PostgreSQL heap tables.
+
+![TPC-C (warehouses = 500)](/docs/img/platform/orioledb-tpc-c-500-warehouse.svg)
+
+## Concepts
+
+### Index-organized tables
+
+OrioleDB uses index-organized tables, where table data is stored in the index structure. This design eliminates the need for separate heap storage, reduces overhead and improves lookup performance for primary key queries.
+
+### No buffer mapping
+
+In-memory pages are connected to the storage pages using direct links. This allows OrioleDB to bypass PostgreSQL's shared buffer pool and eliminate the associated complexity and contention in buffer mapping.
+
+### Undo log
+
+Multi-Version Concurrency Control (MVCC) is implemented using UNDO log. The UNDO log stores previous row versions and transaction information, which enables consistent reads while reducing the need for table vacuuming and minimizing bloat in most cases.
```

Comment:
is it a reduction in need for vacuuming or is vacuum not needed at all?

Reply:
Indeed, vacuum isn't needed. I'll reword this sentence.